### DISCUSSION TITLE
Plot covariate effects on transition probabilities

### DISCUSSION THREAD

**Discussion by willpkay** [bmcclintock/momentuHMM#150] [Q&A]
Hi Brett,

Is there an in-built function in the momentuHMM package (or if not, a workaround) to plot the estimated covariate effects on the transition probabilities? In other words, a plot of transition probability (y-axis) versus covariate effect (x-axis). Ideally I would like to generate predictions of the estimate +- 95% confidence intervals. And if possible, generate separate plots for different mixtures.

Thanks

**Comment by bmcclintock** [comment]
See `?plot.momentuHMM` and examples in the [vignette](https://cran.r-project.org/web/packages/momentuHMM/vignettes/momentuHMM.pdf). For ways to generate your own plots, see #78 and #98.
